TensorFlow2/Segmentation/MaskRCNN/mask_rcnn_main.py

```diff
@@ -111,6 +112,10 @@ def main(argv):
     dllogger.init(backends=[dllogger.JSONStreamBackend(verbosity=dllogger.Verbosity.VERBOSE,
                                                            filename=RUN_CONFIG.log_path)])
 
+    # Set seed to reduce randomness
+    np.random.seed(FLAGS.seed)
+    tf.set_random_seed(RUN_CONFIG.seed)
```

Comment:
Can you also add using the determinism flags and container to your tests? We should see if there are any sources of non-determinism that setting the seeds does not resolve.

Reply:
I have revert the TF changes for now. Currently focusing on PT models.